substrate/frame/broker/src/dispatchable_impls.rs

```diff
@@ -455,4 +455,25 @@ impl<T: Config> Pallet<T> {
 
 		Ok(())
 	}
+  
+  pub(crate) fn ensure_core_availability(
```

Comment:
Can you please add some comment? I don't really get from reading the name what this does, because you don't really pass any core. Or maybe call it `ensure_cores_for_sale`.

Reply:
I think `ensure_cores_for_sale` is good.